test/built-ins/Array/prototype/concat/15.4.4.4-5-b-iii-3-b-1.js

```diff
@@ -15,28 +16,11 @@ Object.defineProperty(Array.prototype, "0", {
 });
 
 var oldArr = [101];
-
 var newArr = Array.prototype.concat.call(oldArr);
 
-var verifyValue = false;
-verifyValue = newArr[0] === 101;
-
-var verifyEnumerable = false;
-for (var p in newArr) {
-  if (p === "0" && newArr.hasOwnProperty("0")) {
-    verifyEnumerable = true;
-  }
-}
-
-var verifyWritable = false;
-newArr[0] = 12;
-verifyWritable = newArr[0] === 12;
-
-var verifyConfigurable = false;
-delete newArr[0];
-verifyConfigurable = newArr.hasOwnProperty("0");
-
-assert(verifyValue, 'verifyValue !== true');
-assert.sameValue(verifyConfigurable, false, 'verifyConfigurable');
-assert(verifyEnumerable, 'verifyEnumerable !== true');
-assert(verifyWritable, 'verifyWritable !== true');
+verifyProperty(newArr, "0", {
+  value: 101,
+  writable: true,
+  enumerable: true,
+  configurable: true,
+});
```

Comment:
lol—awesome :)

Feel free to make this change _everywhere_ that it's appropriate to do so